torch/_inductor/compile_fx.py

```diff
@@ -1158,10 +1159,7 @@ def cudagraphify(
     def run(new_inputs: Sequence[InputType]) -> Any:
         nonlocal compiled_fn
         if compiled_fn is None:
-            with dynamo_utils.dynamo_timed(
-                "cudagraphify",
-                log_pt2_compile_event=True,
-            ), dynamo_utils.preserve_rng_state():
+            with dynamo_utils.preserve_rng_state():
```

Comment:
Pretty sure the placement of this dynamo_timed was wrong. It's confusing because of the closure. I moved it to the cudagraphify call in cudagraph_trees.py. The result looks better to my eyes; see the perfetto trace linked in the test plan for the internal diff D66679369. cc @BoyuanFeng